src/ApiManagement/ApiManagement/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 2.3.0
```

Comment:
please remove this section, no need to provide changelog when no changes in this module.